### PR TITLE
[DOC-296] Add info about deploying manual StatefulSet into a different namespace

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-296]

### CODE REVIEW

File: _includes/v21.2/orchestration/start-cockroachdb-secure.md [hunk 1]
```diff
@@ -17,6 +17,8 @@
       secretName: cockroachdb.node
     ~~~
 
+1. The StatefulSet configuration deploys CockroachDB into the `default` namespace. To use a different namespace, search for `kind: RoleBinding` and change its `subjects.namespace` to the name of the namespace. Otherwise, a `failed to read secrets` error occurs when you attempt to follow the steps in [Initialize the cluster](#initialize-the-cluster).
```
Comment (stbof) [inline _includes/v21.2/orchestration/start-cockroachdb-secure.md:20]: I think a better wording would be one of these:

- change its `subjects.namespace` property 
- change`subjects.namespace`

File: _includes/v22.1/orchestration/start-cockroachdb-secure.md [hunk 2]
```diff
@@ -17,6 +17,8 @@
       secretName: cockroachdb.node
     ~~~
 
+1. The StatefulSet configuration deploys CockroachDB into the `default` namespace. To use a different namespace, search for `kind: RoleBinding` and change its `subjects.namespace` to the name of the namespace. Otherwise, a `failed to read secrets` error occurs when you attempt to follow the steps in [Initialize the cluster](#initialize-the-cluster).
```
Comment (stbof) [inline _includes/v22.1/orchestration/start-cockroachdb-secure.md:20]: Ditto.

**Review by stbof — COMMENTED** [review]
See comment.

File: _includes/v21.2/orchestration/start-cockroachdb-secure.md (hunk 1 above)
Comment (mdlinville) [reply _includes/v21.2/orchestration/start-cockroachdb-secure.md]: Good call. Done.

**Review by stbof — APPROVED** [review]

**Pull request merged**